frontend/src/pages/destructionlist/detail/DestructionListDetail.tsx

```diff
@@ -169,25 +150,58 @@ export async function destructionListUpdateAction({
  * React Router loader.
  */
 export const destructionListDetailLoader = loginRequired(
-  async ({ request, params }: ActionFunctionArgs) => {
-    if (typeof params.uuid === "undefined") return {};
-
+  async ({
+    request,
+    params,
+  }: ActionFunctionArgs): Promise<DestructionListDetailContext> => {
+    const uuid = params.uuid as string;
+    const storageKey = `destruction-list-detail-${uuid}`;
+    const searchParams = Object.fromEntries(new URL(request.url).searchParams);
+
+    // Get reviewers, zaken and zaaktypen.
     const promises = [
-      getDestructionList(params.uuid),
-      getZakenData(request, {
-        not_in_destruction_list_except: String(params.uuid),
-      }),
+      getDestructionList(uuid as string),
       listReviewers(),
+      /*
+       Intercept and ignore 404 due to the following scenario cause by shared `page` parameter:
+
+       - User navigates to destruction list with 1 page of items.
+       - Users click edit button
+       - User navigates to page 2
+       - zaken API with param `in_destruction_list` may return 404.
+      */
```

Comment:
Maybe we need to create an issue for this. I was playing around with it in the browser and it was showing some quirkinesses

Reply:
> Maybe we need to create an issue for this. I was playing around with it in the browser and it was showing some quirkinesses

https://github.com/maykinmedia/open-archiefbeheer/issues/119